action.yml

```diff
@@ -1,55 +1,87 @@
 name: Publish to Chromatic
 author: Chroma Software, Inc.
-description: "Publish your Storybook to Chromatic and run visual regression tests"
+description: 'Publish your Storybook to Chromatic and run visual regression tests'
```

Comment:
Tiny side effect: prettier ran on `action.yml` when I saved, hence the double -> single quotes change. Let me know if you want me to change these back!

Reply:
I'm fine with switching to single quotes (although be aware there's a difference: `'\n'` is interpreted as a literal `\n` string while `"\n"` is interpreted as a newline. In this case, we don't care because it's all single lines.

I do prefer you add quotes to the output descriptions, for consistency.

Reply:
@ghengeveld Fixed!